e2e-tests/trade-finance-manager/cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
```

Comment:
do we need this file?

Reply:
Cypress was failing without it, locally.
Could be due to Cypress v10?

Reply:
in Cypress 10 it's not needed anymore
did you run `npm install` in the root folder?